src/renderer/components/ft-list-video/ft-list-video.vue

```diff
@@ -90,9 +90,7 @@
           <span>{{ channelName }}</span>
         </router-link>
         <template v-if="!isLive && !isUpcoming && !isPremium && !hideViews">
-          <span class="viewCount"><template v-if="channelId !== null"> •</template> {{ parsedViewCount }} </span>
-          <span v-if="viewCount === 1">{{ $t("Video.View").toLowerCase() }}</span>
-          <span v-else>{{ $t("Video.Views").toLowerCase() }}</span>
+          <span class="viewCount"><template v-if="channelId !== null"> •</template>{{ $tc('Global.Counts.View Count', viewCount, {count: parsedViewCount}) }}</span>
```

Comment:
Probably the previous method produces leading spaces and new one does not
Following change tested locally
Also make the long line not so long
```suggestion
          <span class="viewCount">
            <template v-if="channelId !== null"> • </template>
            {{ $tc('Global.Counts.View Count', viewCount, {count: parsedViewCount}) }}
          </span>
```
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/bd66ca79-53a7-424d-bda6-6ad1fcbde41a)